### PR TITLE
split docs

### PR DESCRIPTION
**Pull request by jspaaks**
ref #83

also organized the badges similar to https://github.com/nlesc/template

### CODE REVIEW

**Review by fdiblen — APPROVED** [review]
Refreshing.
:space_invader:

**Comment by jspaaks** [comment]
Very topical emoji

**Review by fdiblen — DISMISSED** [review]
Can you fix the failing checks please?

**Comment by sverhoeven** [comment]
PR #143 and #142 also change the README, make sure mergers dont' loose any text or duplicate.

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -35,6 +35,7 @@ You want to make some kind of change to the code base
 #. (**important**) announce your plan to the rest of the community *before you start working*. This announcement should be in the form of a (new) issue;
 #. (**important**) wait until some kind of consensus is reached about your idea being a good idea;
 #. if needed, fork the repository to your own Github profile and create your own feature branch off of the latest master commit. While working on your feature branch, make sure to stay up to date with the master branch by pulling in changes, possibly from the 'upstream' repository (follow the instructions `here <https://help.github.com/articles/configuring-a-remote-for-a-fork/>`__ and `here <https://help.github.com/articles/syncing-a-fork/>`__);
+#. if cloned locally then perform a `development installation <README.dev.rst#development-install>`;
```
Comment (jspaaks) [inline CONTRIBUTING.rst:38]: I think you need a trailing underscore here

```suggestion
#. if cloned locally then perform a `development installation <README.dev.rst#development-install>`_;
```

**Comment by jspaaks** [comment]
I can't approve|request cahnges because I started this PR, but looks good to me except for that small thing above.

**Comment by sverhoeven** [comment]
Thanks for reviewing

**Pull request merged**